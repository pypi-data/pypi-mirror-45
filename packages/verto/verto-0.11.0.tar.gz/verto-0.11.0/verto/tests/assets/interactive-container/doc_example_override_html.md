{interactive slug="binary-cards" type="whole-page" parameters="digits=5&start=BBBBB" thumbnail="binarycards.png" text="true"}

Binary Cards Interactive

{interactive end}
