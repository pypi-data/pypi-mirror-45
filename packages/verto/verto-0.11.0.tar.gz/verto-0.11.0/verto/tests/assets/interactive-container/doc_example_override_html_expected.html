<a class="btn btn-interactive-link" href="{% url 'interactive' slug='binary-cards' %}?digits=5&amp;start=BBBBB">
<div class="btn-interactive-whole-page-text">
Binary Cards Interactive
</div>
</a>
